hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/AdditionalStudiesTransformations.scala

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.monster.dap.dog
+
+import org.broadinstitute.monster.dap.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.HlesDogFutureStudies
+
+object AdditionalStudiesTransformations {
+
+  /**
+    * Parse all future-studies health fields out of a raw RedCap record,
+    * injecting them into a partially-modeled dog record.
+    */
+  def mapFutureStudies(rawRecord: RawRecord): HlesDogFutureStudies = {
+    val init = HlesDogFutureStudies.init()
+
+    val transformations = List(
+      mapFutureStudiesFields _
+    )
+
+    transformations.foldLeft(init)((acc, f) => f(rawRecord, acc))
+  }
+
+  /**
+    * Parse primary care vet and participation likelihood fields out of a
+    * raw RedCap record, injecting them into a partially-modeled dog record.
+    */
+  def mapFutureStudiesFields(
+    rawRecord: RawRecord,
+    dog: HlesDogFutureStudies
+  ): HlesDogFutureStudies = {
+    val pcVetConsent = rawRecord.getBoolean("fs_pcvet_consent")
+    dog.copy(
+      fsPrimaryCareVeterinarianExists = rawRecord.getOptionalBoolean("fs_pcvet"),
```

Comment:
I originally had this split out into two functions - one for primaryCareVet fields and one for participationLikelihood fields. I later collapsed them into the same function since it was such a small number of fields. Is it okay for them all to be processed in the same function?

Reply:
Yeah, I think it's small enough to be reasonable